frontend/src/container/WorkspaceRecentActivities.jsx

```diff
@@ -146,7 +145,7 @@ WorkspaceRecentActivities.propTypes = {
   handleTlm: PropTypes.func.isRequired,
   onRefreshClicked: PropTypes.func.isRequired,
   onCopyLinkClicked: PropTypes.func.isRequired,
-  workspaceId: PropTypes.string.isRequired
+  workspaceId: PropTypes.number.isRequired
```

Comment:
we don't have a clear type for workspace id, it is sometime an int and sometime a string. Best way to be sure to never have warning can be to allow both from propTypes

Reply:
fix